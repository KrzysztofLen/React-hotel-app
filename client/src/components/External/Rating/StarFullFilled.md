Star example:

```js
<Star fill="#BA265D"/>
```