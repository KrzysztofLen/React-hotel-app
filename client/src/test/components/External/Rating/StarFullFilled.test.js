import React from 'react';
import {shallow} from 'enzyme';
import StarFullFilled from '../../../../components/External/Rating/StarFullFilled';

test('it should render star correctly', () => {
	const fillStar = "#BA265D";
	const wrapper = shallow(<StarFullFilled fill={fillStar}/>);
	expect(wrapper.debug()).toMatchSnapshot();
});