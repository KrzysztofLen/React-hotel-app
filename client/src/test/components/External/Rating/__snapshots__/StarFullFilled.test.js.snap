// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render star correctly 1`] = `
"<svg version=\\"1.1\\" xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"20\\" height=\\"20\\" viewBox=\\"0 0 20 20\\" fill=\\"#BA265D\\">
  <title>
    star
  </title>
  <path d=\\"M10 1.3l2.388 6.722h6.412l-5.232 3.948 1.871 6.928-5.439-4.154-5.438 4.154 1.87-6.928-5.233-3.948h6.412l2.389-6.722z\\" />
</svg>"
`;
